Models/SunglassesKhronos/metadata.json

```diff
@@ -0,0 +1,47 @@
+{
+    "version": 2,
+    "legal": [
```

Comment:
Should there be explicit copyright info for all the images (especially photos) used in the readme?

Reply:
I'm not sure. We have photos in other readmes, without explicit licensing. For example https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/AnisotropyBarnLamp

Reply:
Also https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/GlamVelvetSofa#reference-photos
and https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/SheenChair#khr_materials_sheen
and https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/StainedGlassLamp#reference-photos